zokrates_core/src/absy/from_ast.rs

```diff
@@ -0,0 +1,515 @@
+use absy;
+use imports;
+use types::Type;
+use zokrates_field::field::Field;
+use zokrates_pest_ast as pest;
+
+impl<'ast, T: Field> From<pest::File<'ast>> for absy::Prog<'ast, T> {
```

Comment:
Consideration not blocking a merge now:
From a separation of concerns perspective, it could be cleaner to expose a to_absy in the zokrates_pest_ast instead of having the from zokrates_pest_absy here.